### PR TITLE
Add optional argument to customize preview comment

### PR DESCRIPTION
**Pull request by jbusecke**
Thank you so much for these super useful actions. 

This PR adds an optional input argument for the preview comment action to enable more customized reuse of the action (outside of the ProjPythia specific context).  This should be backwards compatible as far as I can tell.

### CODE REVIEW

**Comment by brian-rose** [comment]
Hi @jbusecke!

This looks like a useful addition, and agreed should be completely backwards compatible. We haven't yet implemented proper CI tests for this repo (see #6) so it's a bit of a guessing game, but this looks pretty clear so I'll merge it.

File: .github/workflows/preview-comment.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ on:
         description: 'SHA of the commit associated with this Pull Request'
         required: true
         type: string
+      site_name: 
+        description: 'Name of the website in comment'
+        required: false
+        default: "The Cookbook"
```
Comment (brian-rose) [inline .github/workflows/preview-comment.yaml:17]: I think this needs
```
type: string
```
unless that is interpreted implicitly from the default value?

Comment (jbusecke) [reply .github/workflows/preview-comment.yaml:17]: Sounds good to me (but I am also not very knowledgeable in this regard hehe).

**Review by brian-rose — APPROVED** [review]

**Comment by brian-rose** [comment]
I'll merge this, and on the off chance that it breaks something, we'll circle back and fix it.

Thanks for the contribution @jbusecke!

**Pull request merged**